hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -222,19 +226,21 @@ public void createTable(String table, List<byte[]> cfs) throws IOException {
                                                              .build());
         }
         tdb.setCoprocessor(COPROCESSOR_AGGR);
-        try(Admin admin = this.hbase.getAdmin()) {
+        try (Admin admin = this.hbase.getAdmin()) {
             admin.createTable(tdb.build());
         }
     }
 
-    public void createPreSplitTable(String table, List<byte[]> cfs, short numOfPartitions) throws IOException {
+    public void createPreSplitTable(String table, List<byte[]> cfs, 
+                                    short numOfPartitions) throws IOException {
         TableDescriptorBuilder builder = TableDescriptorBuilder.newBuilder(
                 TableName.valueOf(this.namespace, table));
         for (byte[] cf : cfs) {
             builder.setColumnFamily(ColumnFamilyDescriptorBuilder.newBuilder(cf)
                     .build());
```

Comment:
align or not to divide?

Reply:
No rules will be hit.
The below line is indented more than 4 blankspace. 
And we setted forceStrictCondition = false, so more than 4 space is ok.

https://checkstyle.sourceforge.io/config_misc.html#Indentation.

there isn't any rules about align in all checkstyle rules

https://checkstyle.sourceforge.io/checks.html